apps/system-e2e/README.md

```diff
@@ -193,7 +193,7 @@ You will need a few things to set up your test so it can run with mountebank.
 
 Now that you are set up. You need to run a couple of commands.
 
-- In your terminal run `yarn infra render-local-env --service=service-portal --service=api` .
+- In your terminal run `yarn cli render-local-env --service=service-portal --service=api` .
```

Comment:
Isn't this unnecessary? Your changes seem to be (mostly) backwards-compatible.

Reply:
`yarn infra` just runs install now instead of installing _and_ invoking the cli.